docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -35,9 +35,19 @@ Repeat this configuration for other networks that need to use different subnets
 . Apply the new configuration to every {OpenStackShort} node:
 +
 ----
-$ openstack overcloud node provision
---stack <stack>
---network_config
+(undercloud)$ openstack overcloud network provision \
+ [--templates <templates_directory> \]
+ --output  <deployment_file> \
+ /home/stack/templates/<networks_definition_file>
+----
++
+----
+(undercloud)$ openstack overcloud node provision \
+ [--templates <templates_directory> \]
+ --stack <stack> \
+ --network-config \
+ --output <deployment_file> \
+ /home/stack/templates/<node_definition_file>
```

Comment:
We also need the list that describes the values to be replaced from step 10 in [1].
I can do this in another PR if you want. But please confirm that the items below the command in step 10[1] are relevant for this step in "Configuring new subnet ranges".

[1] https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/installing_and_managing_red_hat_openstack_platform_with_director/assembly_provisioning-and-deploying-your-overcloud#proc_provisioning-bare-metal-nodes-for-the-overcloud_ironic_provisioning

Reply:
When running `overcloud node provision` - we also need to toggle the `network_config_update` for each role in the `<node_definition_file>`. Ref the [4.11. Adding a new leaf to a spine-leaf deployment](https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/configuring_spine-leaf_networking/assembly_configuring-the-overcloud#proc_add-new-leaf_spine-leaf)[1] doc, step 9.

I think you can add that in the follow up as well.

https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/configuring_spine-leaf_networking/assembly_configuring-the-overcloud#proc_add-new-leaf_spine-leaf